modules/rules_spring/2.4.2/patches/module_dot_bazel.patch

```diff
@@ -0,0 +1,26 @@
+--- MODULE.bazel
++++ MODULE.bazel
+@@ -1,6 +1,19 @@
+ module(
+-  name = "rules_spring",
+-  version = "2.4.2",
+-  repo_name = "rules_spring",
+-  compatibility_level = 1,
++    name = "rules_spring",
++    version = "2.4.2",
++    compatibility_level = 1,
+ )
++
++bazel_dep(name = "rules_python", version = "0.40.0")
++
++# For Dupe Class checking support
++python = use_extension("@rules_python//python/extensions:python.bzl", "python")
++python.toolchain(
```

Comment:
Is it really necessary to pin it to 3.11?

Reply:
I can try in 2.5.1 to do that. After 2.4.2 is added here, I will need to add 2.5.x. I want a 2.4.x release in BCR because I want teams to be able to have another option in case 2.5.x doesn't work for them.

Latest:
https://github.com/salesforce/rules_spring/blob/main/MODULE.bazel